src/generic/generic-component-tag/generic-component-tag.scss

```diff
@@ -8,14 +8,20 @@
 
 @mixin ecl-tag() {
   .ecl-tag {
+    margin: 0;
+  }
+
+  .ecl-tag__container {
     align-items: center;
     display: flex;
-    margin: 0;
+    flex-wrap: wrap;
+    margin-top: - map-get($ecl-spacing, 'xxxs');
```

Comment:
This is just a proposal. Because `.ecl-tag__item` can now wrap to the size of the container, I had to add a margin-top for spacing. That forced me to add a negative margin-top for the newly created container. For me it's like adding a margin to the component itself which is against our `margin: 0` policy. Screenshots need to be updated if this proposal is accepted.